docs/troubleshooting-guide/proxies.md

```diff
@@ -0,0 +1,33 @@
+If you're experiencing problems where saving questions/dashboard sometimes fails or Metabase only loads a blank page, then it might be caused by a proxy.
+
+A proxy could include other functions like web application firewall (WAF), content optimization or cache.
+
+Examples of proxies that are known to cause issues with Metabase:
+
+- Cloudflare's Rocket Loader and WAF
+- Azure's WAF
+- PageSpeed module for i.e Apache
+- Some anti-virus browser extensions/add-ons
+
+## Specific Problems
+
+### Saving questions/dashboards fails
+
+If saving questions/dashboards fails with the button changing to `Save Failed` or perhaps with the error `Sorry you do not have permission to see that`, then it might be caused by either a WAF like Cloudflare or Azure.
```

Comment:
```suggestion
If saving questions/dashboards fails with the save button changing to "Save Failed," or perhaps with the error, "Sorry you do not have permission to see that," this might be caused by either a WAF like Cloudflare or Azure.
```